functions/db/import_histograms.fn.js

```diff
@@ -0,0 +1,95 @@
+const admin = require('firebase-admin');
+const { BigQuery } = require('@google-cloud/bigquery');
+const functions = require('firebase-functions');
+const { murmurHash128 } = require('murmurhash-native');
+
+try {
+  admin.initializeApp();
+} catch (e) {
+  console.log(e);
+}
+
+async function readAggregates(channel, version) {
+  const bigquery = new BigQuery();
+  const query = `
+    SELECT *
+    FROM \`moz-fx-data-derived-datasets.analysis.client_probe_counts\`
+    WHERE
+      channel = @channel
+      AND app_version = @app_version
+      AND metric = @metric
+      AND os = @os
+  `;
+  const options = {
+    query,
+    location: 'US',
+    params: {
+      channel,
+      app_version: version,
+      metric: 'scalar_content_browser_usage_plugin_instantiated',
+      os: 'Windows_NT'
+    }
+  };
+  const [rows] = await bigquery.query(options);
```

Comment:
Small thing, but in my experience, `const [ rows ]` is more conventional and may be easier to read by some people as a result. I did a double-take on this line.

Reply:
Also, as we talked about in IRC, this just grabs the first element from the array that `bigquery.query()` returns and assigns it the name `rows`.